docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -48,12 +49,21 @@ Microsimulation models incorporate relevant characteristics, including:
 - Adverse events
 - Mortality rate/life expectancy 
 
-.. image:: microsim_example_table_2.PNG
+The below tables provide a simplified illustration of what is included in a Vivarium microsimulation. Both tables are based on a microsimulation of 
+antenatal nutritional supplementation effects on birthweight and child-growth failure. 
 
-The above table provides an illustration of the simulant attributes that might be included in a Vivarium microsimulation. This particular example is pulled from a microsimulation of 
-antenatal nutritional supplementation effects on birthweight and child-growth failure. As you can see, the simulant listed here has attributes covering demography, risk factors (e.g., 
-*did the mother/birthing person received iron and folic acid (IFA) supplementation during pregnancy?*), causes (e.g., *what is the probability of child simulant having protein energy malnutrition (PEM)?*),
-and mortality (e.g., *is the simulant alive or dead at this given timestep?*).
+.. image:: microsim_example_table_3.PNG
+
+This first table outlines some of the simulant attributes we can include in our microsimulations. The simulants listed here have demographic attributes
+such as sex and location, as well as risk factors (e.g., *did the mother/birthing person received iron and folic acid (IFA) supplementation during pregnancy?*).
+
+.. image:: microsim_example_table_4.PNG
```

Comment:
I think this is clear enough, but I do think it would be slightly clearer if these tables were split vertically for different timesteps. I was imagining a break with "... two days pass..." in between, but I'm not sure how you would do that in Excel (merged cells?). Likewise, I think a little bit of space between the two scenarios would be nice.

Reply:
Ah yeah, that makes sense! I can give that a try!

Reply:
Just tried to make those changes - lmk if they don't look quite right still!

Reply:
Looks great to me!